comfy_cli/command/custom_nodes/command.py

```diff
@@ -824,92 +859,6 @@ def display_all_nodes():
     )
 
 
-@app.command(
-    "registry-install",
-    help="Install a node from the registry",
-    hidden=True,
-)
-@tracking.track_command("node")
-def registry_install(
```

Comment:
`registry_install` -> we would need to update the commands inside the registry_web repository https://github.com/Comfy-Org/registry-web/blob/23948cfe4faa73f640a48139479a718a2d561ccd/components/nodes/NodeDetails.tsx#L209. 

This should be replaced with a new command. `node install <node_id>@<version>`